arb_os/arbsys.mini

```diff
@@ -499,18 +499,18 @@ impure func arbAddressTable_decompress(topFrame: EvmCallFrame, calldata: ByteArr
         )) {
             evmOp_return(0, 64);
         } else {
-            evmOp_revert_knownCodePc(address(102), 162, 0, 0);
+            evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 162, 0, 0);
         }
     } else {
         // input was too short
-        evmOp_revert_knownCodePc(address(102), 163, 0, 0);
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 163, 0, 0);
     }
 }
 
 impure func arbAddressTable_compress(topFrame: EvmCallFrame, calldata: ByteArray) { // (address addr) -> bytes
 // compress addr, based on the current address table contents, and return the resulting buffer
     if (bytearray_size(calldata) != 36) {
-        evmOp_revert_knownCodePc(address(102), 100, 0, 0);
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), const::Address_ArbSys, 0, 0);
```

Comment:
This seems like another incorrect use of `const::Address_ArbSys`

Reply:
Fixed.